uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js

```diff
@@ -157,7 +176,6 @@ define([
        */
       vm._initRequest = function () {
         var attributes = vm._initRequestAttributes();
-
```

Comment:
Please leave this line since it is between var assignment and property assignment.